From 8462eb4536d0fa1e0ed704bc205b288341b66775 Mon Sep 17 00:00:00 2001
From: Simon Fels <simon.fels@canonical.com>
Date: Wed, 29 Mar 2017 17:33:34 +0200
Subject: [PATCH] cmd: add poky to the list of distros which don't support
 reexec

'poky' is the std. release id for the Yocto reference distribution. This
wont cover all Yocto based product as Yocto is a meta distribution so
users are fine to change the release id to whatever they want. However
this clearly takes care just about our reference Yocto system.
---
 cmd/cmd.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/cmd.go b/cmd/cmd.go
index c70b6cc..c681b39 100644
--- a/cmd/cmd.go
+++ b/cmd/cmd.go
@@ -62,7 +62,7 @@ func ExecInCoreSnap() {
 
 	// can we re-exec? some distributions will need extra work before re-exec really works.
 	switch release.ReleaseInfo.ID {
-	case "fedora", "centos", "rhel", "opensuse", "suse":
+	case "fedora", "centos", "rhel", "opensuse", "suse", "poky":
 		logger.Debugf("re-exec not supported on distro %q yet", release.ReleaseInfo.ID)
 		return
 	}
-- 
2.7.4

